### ISSUE TITLE
[Code Addition Request]: Face Mask Detection

### ISSUE THREAD

**Issue by ramana2074** [UTSAVS26/PyVerse#1167]
### Have you completed your first issue?

- [x] I have completed my first issue

### Guidelines

- [x] I have read the guidelines
- [x] I have the link to my latest merged PR

### Latest Merged PR Link

https://github.com/UTSAVS26/PyVerse/pull/1127https://github.com/UTSAVS26/PyVerse/pull/1127
#1127 

### Project Description

Using a pre-trained PyTorch model to detect whether a person in an image is wearing a face mask, wearing it incorrectly, or not wearing a mask at all.

### Full Name

Venkata Ramana Billana

### Participant Role

SWOC Contributor

**Comment by ritik4ever** [comment]
I want to work on this issue

**Comment by UTSAVS26** [comment]
> I want to work on this issue 

Just wait for the official kickoff of the program them you can raise issues and comment on existing issues. 

And I am closing all the issues that have been raised till now and gonna raise few issues for each level for each domain soon.
